sat/cli/bootprep/input/session_template.py

```diff
@@ -211,6 +172,150 @@ def create(self, dumper=None):
             raise SessionTemplateCreateError(f'Failed to create session template: {err}')
 
 
+class InputSessionTemplateV1(InputSessionTemplate):
```

Comment:
I agree. I was thinking about this while writing the code, but when I started to try to put more meaning into the names, it did feel like they were getting long and hard to parse. Your suggestions are pretty good though. I'll consider it further.

Reply:
I'm going to ignore this for now since we're on a tight schedule, but I'm definitely open to a rename in the future. It doesn't affect anything user-facing, so it can be changed easily later.

Reply:
Sounds good. It definitely isn't super high priority at the moment.